api/envoy/config/listener/v3/listener.proto

```diff
@@ -248,4 +249,9 @@ message Listener {
   // Configuration for :ref:`access logs <arch_overview_access_logs>`
   // emitted by this listener.
   repeated accesslog.v3.AccessLog access_log = 22;
+
+  // UdpWriterProto in Listener Config
+  // This field specifies the udp_writer_name, e.g. "udp_default_writer" or
+  // "udp_gso_batch_writer", that can be used to instantiate udp_writer.
+  UdpWriterConfig udp_writer_config = 23;
```

Comment:
You don't have to define another message for a new extension type, just use `TypedExtensionConfig`.

Reply:
Thanks for the suggestion Lizan! I have updated the file to now use `TypedExtensionConfig` for `udp_writer_config`.